ansible/playbooks/create-meter.yaml

```diff
@@ -44,27 +44,44 @@
       shell: pwgen 10 1 > /energy/client_pass
       args:
         creates: /energy/client_pass
+    - name: clean old parity accounts
+      shell: rm -rf /energy/parity/datadir/keys/*
     - name: create new account
       shell: parity --chain /energy/parity/res/genesis.json --keys-path /energy/parity/datadir//keys account new --password /energy/client_pass > /energy/client_addr
     - name: get money to the account
       shell: curl -X POST -d `cat /energy/client_addr` http://146.169.47.73:6000/money
       ignore_errors: yes
     - name: write the account to config.toml
-      shell: echo "[account]" >> /energy/parity/res/config.toml && echo "  unlock = ['`cat /energy/client_addr`']" >> /energy/parity/res/config.toml && echo "  password = [\"/energy/client_pass\"]" >> /energy/parity/res/config.toml
+      shell: echo "[account]" >> /energy/parity/res/config.toml
+             && echo "  unlock = ['`cat /energy/client_addr`']" >> /energy/parity/res/config.toml
+             && echo "  password = [\"/energy/client_pass\"]" >> /energy/parity/res/config.toml
+    - name: install truffle
+      shell: npm install -g truffle
+    - name: build contracts
+      shell: rm -rf /energy/truffle/build && cd /energy/truffle && truffle compile && truffle build
     - name: reload systemd
       shell: systemctl daemon-reload
+    - name: Disable ParityEnergy service if running
+      shell: systemctl disable parity-energy && systemctl stop parity-energy
```

Comment:
stop then disable?

Reply:
Also, if disable fails, it cannot be stopped.

Reply:
Doesn't matter, disable just kills symlinks

Reply:
ok